### ISSUE TITLE
Error text encoding in windows

### ISSUE THREAD

**Issue by DGolubets** [lukeed/taskr#136]
The output in my console I get in case of any error:

```
~~[~3~2~m~s~t~a~c~k~:~ ~~[~3~9~m~
~ ~ ~~[~3~2~m~-~ ~~[~3~9~m~S~y~n~t~a~x~E~r~r~o~r~:~ ~C~:~/~U~s~e~r~s~/~d~g~o~l~u~/~P~r~o~j~e~c~t~s~/~n~o~d~e~-~r~e~a~c~t~-~w~e~b~/~F~l~y~f~i~l~e~.~j~s~:~ ~U~n~e~x~p~e~c~t~e~d~ ~t~o~k~e~n~ ~(~1~2~:~3~5~)~
~ ~ ~~[~3~2~m~-~ ~~[~3~9~m~~[~0~m~ ~ ~1~0~ ~|~ ~~[~3~6~m~f~u~n~c~t~i~o~n~~[~3~9~m~ ~c~l~e~a~n~~[~9~4~m~~[~1~m~(~~[~2~2~m~~[~3~9~m~o~p~t~i~o~n~s~~[~1~m~,~~[~2~2~m~ ~c~a~l~l~b~a~c~k~~[~9~4~m~~[~1~m~)~~[~2~2~m~~[~3~9~m
~ ~~[~3~2~m~{~~[~3~9~m~
```

Windows 10

**Comment by ghost** [comment]
Fly has been known to work on Windows (hehe), but other than asking you for details about your console, shell environment and support for ANSI escape sequences (colors, etc). I can't help much.

Perhaps @iamstarkov could have something to say?

**Comment by DGolubets** [comment]
I tried in Cmd and Powershell.
Also node errors are fine. So it must be Fly or something it uses.
And my node is 4.2.6

Edit:
it seems to work fine from Mingw.

Edit2: just saw a previous issue about the same:  #101

**Comment by lukeed** [comment]
Closing & marking as duplicate; however I'll be sure to ping you if/when this is resolved in #101.
